bigchaindb/consensus.py

```diff
@@ -31,8 +32,8 @@ def validate_transaction(bigchain, transaction):
             Descriptive exceptions indicating the reason the transaction failed.
             See the `exceptions` module for bigchain-native error classes.
         """
-        raise NotImplementedError
```

Comment:
Using ABC classes you dont need to
They return a

```
TypeError: Can't instantiate abstract class B with abstract methods f
```

if method `f` wasnt concrete